ecommerce/views_test.py

```diff
@@ -693,6 +720,29 @@ def test_patch_basket_update_active_inactive_product(basket_client, basket_and_c
     assert resp.status_code == status.HTTP_200_OK
 
 
+def test_patch_basket_update_active_inactive_product_readable_id(
+    basket_client, basket_and_coupons
+):
+    """ Test that inactive product id is rejected with no changes to basket but not the active ones. """
+    product = ProductVersionFactory.create().product
+    product.is_active = False
+    product.save()
```

Comment:
```python
product_version = ProductVersionFactory.create(product__is_active=False)
product = product_version.product
```

Reply:
This can also be optimized by using that way.
`product = ProductVersionFactory.create(product__is_active=False).product`